demos/top-app-bar.html

```diff
@@ -119,25 +117,107 @@
+        </p>
+        <p class="demo-paragraph">
+          Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est.
+        </p>
+
       </div>
     </main>
 
+    <div class="demo-controls-container">
+      <h3>Demo Controls</h3>
+      <div>
+        <input type="checkbox" id="rtl-checkbox"/>
+        <label for="rtl-checkbox">RTL</label>
+      </div>
+      <div>
+        <input type="checkbox" id="short-checkbox"/>
+        <label for="short-checkbox">Short</label>
+      </div>
+      <div>
+        <input type="checkbox" id="no-right-item-checkbox"/>
+        <label for="no-right-item-checkbox">No Right Item</label>
+      </div>
+    </div>
+
     <script src="/assets/common.js" async></script>
     <script src="/assets/material-components-web.js" async></script>
     <script type="text/javascript">
-    demoReady(function() {
-      var appBarEl = document.getElementById('demo-top-app-bar');
-      var appBar = mdc.topAppBar.MDCTopAppBar.attachTo(appBarEl);
+      demoReady(function() {
+        var appBarEl = document.getElementById('demo-top-app-bar');
+        var rightItemEl = document.getElementById('right-item');
+        var rightSection = document.getElementById('iconSection');
+        var drawerEl = document.querySelector('.mdc-drawer');
 
-      var drawerEl = document.querySelector('.mdc-drawer');
-      var MDCTemporaryDrawer = mdc.drawer.MDCTemporaryDrawer;
-      var drawer = new MDCTemporaryDrawer(drawerEl);
+        var drawer = new mdc.drawer.MDCTemporaryDrawer(drawerEl);
+        var appBar = mdc.topAppBar.MDCTopAppBar.attachTo(appBarEl);
 
-      appBarEl.addEventListener('MDCTopAppBar:nav', function() {
-        drawer.open = true;
-      });
+        var shortCheckbox = document.getElementById('short-checkbox');
+        var noRightItemCheckbox = document.getElementById('no-right-item-checkbox');
+        var rtlCheckbox = document.getElementById('rtl-checkbox');
 
-    });
+        appBarEl.addEventListener('MDCTopAppBar:nav', function() {
+          drawer.open = true;
+        });
+
+        shortCheckbox.addEventListener('change', function() {
+          this.checked ? appBarEl.classList.add('mdc-top-app-bar--short') : appBarEl.classList.remove('mdc-top-app-bar--short');
```

Comment:
`appBarEl.classList[this.checked ? 'add' : 'remove'](...);`

(It's awkward to use a ternary for an entire statement.)